thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -79,46 +77,6 @@ const newWowList = wowOptions.map((item) => {
   return { name: item, isActive: false };
 });
 
-/**
```

Comment:
Converting these fetching functions to concurrency tasks. (to be ported into services later on)